File: engine/model/src/main/scala/net/server/channel/packet/command/admin/SummonMonsterPacket.scala
package net.server.channel.packet.command.admin

import net.server.MaplePacket

class SummonMonsterPacket( private var _mode: Byte,  private var _mobId: Int,  private var _quantity: Int) extends BaseAdminCommandPacket(_mode = _mode) {
     def mobId: Int = _mobId
     def quantity: Int = _quantity
}

File: engine/engine-model/src/main/java/net/server/channel/packet/command/admin/SummonMonsterPacket.java
package net.server.channel.packet.command.admin;

public class SummonMonsterPacket extends BaseAdminCommandPacket {
   private final Integer mobId;

   private final Integer quantity;

   public SummonMonsterPacket(Byte mode, Integer mobId, Integer quantity) {
      super(mode);
      this.mobId = mobId;
      this.quantity = quantity;
   }

   public Integer mobId() {
      return mobId;
   }

   public Integer quantity() {
      return quantity;
   }
}
